flake.nix

```diff
@@ -47,7 +47,14 @@
                 inherit nmdSrc;
               };
 
-              legacyPackages.makeNixvim = import ./wrappers/standalone.nix pkgs (modules pkgs);
+              legacyPackages = rec {
+			    makeNixvimWithModule = import ./wrappers/standalone.nix pkgs modules;
+			  	makeNixvim = configuration: makeNixvimWithModule { 
+					module = {
+						config = configuration;
+					};
+				};
+			   };
```

Comment:
Looks alright, can you fix the indentation though? (You're using tabs instead of 2 spaces like in the rest of the file!)

Reply:
Fixed it! Yeah I forgot my neovim was indenting with tabs